.gitignore

```diff
@@ -0,0 +1,4 @@
+webapp/node_modules
+webapp/.cache
+webapp/dist
+.idea
```

Comment:
Could you please also add [target/](https://github.com/holounic/STEP/blob/master/.gitignore) which as far as I know is a folder Maven uses for build artifacts.

Reply:
nit: [Add a newline at the end of the file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file). Some tools in the Unix world get confused if a nonempty text file does not end with a newline.

Reply:
Updated with commits 47515f9 and 8880282